sense/finetuning.py

```diff
@@ -399,7 +400,8 @@ def run_epoch(data_loader, net, criterion, optimizer=None, use_gpu=False,
     top1 = np.mean(epoch_labels == epoch_top_predictions)
     loss = running_loss / len(data_loader)
 
-    cnf_matrix = confusion_matrix(epoch_labels, epoch_top_predictions)
+    cnf_matrix = confusion_matrix(epoch_labels, epoch_top_predictions,
+                                  labels=range(min(epoch_labels), max(epoch_labels) + 1))
```

Comment:
Would it be safer to go from 0 to `len(label_names_temporal)` just in case the epoch doesn't contain an example for one of those endpoints?